### PR TITLE
Update wallgen.py

### PR DESCRIPTION
**Pull request by priyamharsh14**
#49 Added the option to set up wallpaper after generating image (for windows system)

### CODE REVIEW

**Comment by SubhrajitPrusty** [comment]
- Can you move the code to a different file inside tools directory?
- Make the function as a Click CLI option, ie do not ask user for input

**Comment by SubhrajitPrusty** [comment]
also send the PR to dev branch please

File: wallgen.py [hunk 1]
```diff
@@ -1,12 +1,26 @@
...
 from tools.shapes import *
 
 CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
 
+def set_wallpaper(file_name):
+	if sys.platform == 'win32':
+		ch = input(f"Do you want to set it as your wallpaper? (y/N): ")
+		if ch.lower().strip()=='y':
+			try:
+				ctypes.windll.user32.SystemParametersInfoW(20, 0, os.getcwd()+"\\"+file_name, 3)
```
Comment (SubhrajitPrusty) [inline wallgen.py]: This line won't work if the user uses the -n flag to set a directory appended with a file path.

**Comment by priyamharsh14** [comment]
Ok.. I will review the code again acc to your specifications

**Pull request closed without merging**